### ISSUE TITLE
Show First Detection Image on Transient Source Page

### ISSUE THREAD

**Issue by gijzelaerr** [transientskp/banana#43]
For reported transients it would be nice to have the first image it was detected in there on the transient page with the transient marked. This comes from feedback of Tom and Gosia using Banana and I agree that this would be a useful inclusion.

original issue:
https://support.astron.nl/lofar_issuetracker/issues/5409

**Comment by gijzelaerr** [comment]
ok, no response for a month now, probably not that important. Going to release 2.0, moving this to 2.1

**Comment by timstaley** [comment]
Yep fine, sorry I couldn't find time for this - I think it's OK as is.

**Comment by gijzelaerr** [comment]
so, i'm not sure if this is a valid issue anymore. We don't really have a 'transient' view anymore, you are just redirected to the lightcurve page. There, you can see which source triggered the event and click trough to the image page.

@timstaley what do you think?
